File: apollo/src/main/java/io/muun/apollo/domain/errors/AmountTooSmallError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class AmountTooSmallError extends UserFacingError {

    public AmountTooSmallError() {
        super(UserFacingErrorMessages.INSTANCE.amountTooSmall());
    }
}

File: apollo/src/main/java/io/muun/apollo/domain/errors/AmountTooSmallError.kt
package io.muun.apollo.domain.errors


import io.muun.apollo.external.UserFacingErrorMessages

class AmountTooSmallError(amountInSats: Long):
    UserFacingError(UserFacingErrorMessages.INSTANCE.amountTooSmall()) {

    init {
        metadata["amountInSats"] = amountInSats
    }

}
